CRM/Utils/Date.php

```diff
@@ -414,6 +439,8 @@ public static function customFormat($dateString, $format = NULL, $dateParts = NU
       }
 
       $date = [
+        '%A' => $fullWeekdayNames[$dayInt] ?? NULL,
```

Comment:
I think this PR is otherwise good but doesn't this change the output for anywhere/anyone currently using `%A`? Before it was AM/PM.

Reply:
it would but grepping in the universe I am not finding references to anyone calling this function and passing in "%A" so I think we are safe to fix it and looking at the git / svn history it was added here https://github.com/civicrm/civicrm-svn/commit/30694cd87776780b111ccb36dd80524d3df158b3 . It looks like it might have been used as part of custom field formatting but that seems long gone now

Reply:
I see your point, but it's used in message templates (via crmDate) which isn't a greppable thing. It's also used for these https://dmaster.demo.civicrm.org/civicrm/admin/setting/date?reset=1 (here's [one example](https://github.com/civicrm/civicrm-core/blob/2010843411d66c9dbba598ce8776a81cefaf975c/CRM/Contribute/Page/ContributionRecurPayments.php#L108)). Although, while I do know of sites that change those, they would usually use %P which is the same and already there. In that svn link it's kind of confusing why they would add %A when right above where they added it it already had %P. But there's still message templates, but yes it might be rare. Perhaps an upgrade message? Wouldn't have to be in this PR.

Is time travel a feature in civi yet? Maybe part of api4? Can we go back in time and fix the svn? I also had a need for time travel the other day.

Reply:
@demeritcowboy The default templates are greppable and in none of the defaults they are using `%A` that gets passed through and the help text in that date format is pushing towards %P rather than %A but yes I can see that I think an upgrade status message would be sensible